client-admin/src/components/conversation-admin/conversation-config.js

```diff
@@ -144,6 +144,10 @@ class ConversationConfig extends React.Component {
           Customize the user interface
         </Heading>
 
+        <CheckboxField field="priority_type" label="Weight">
```

Comment:
label should not be called 'weight' here.

Reply:
Also, let's rename 'priority_type' to 'importance_enabled' and keep the terminology consistent across the code.